src/lib/php/Dao/ClearingDao.php

```diff
@@ -599,6 +599,15 @@ public function isDecisionTBD($uploadTreeId, $groupId)
     return ($latestDec['decision_type'] == DecisionTypes::TO_BE_DISCUSSED);
   }
 
+  public function isDecisionDNU($uploadTreeId, $groupId)
+  {
+    $sql = "SELECT decision_type FROM clearing_decision WHERE uploadtree_fk=$1 AND group_fk = $2 ORDER BY date_added DESC LIMIT 1";
```

Comment:
There is no index on `clearing_decision.date_added`. So it will be better to sort with `clearing_decision.clearing_decision_pk` as they both are related.

Also, will it be okay to break the lines to keep the 80 characters line limit?